mscore/exportxml.cpp

```diff
@@ -4929,7 +4929,7 @@ void ExportMusicXml::print(const Measure* const m, const int partNr, const int f
                         newThing = " new-page=\"yes\"";
                   }
             else if (preferences.musicxmlExportBreaks() == MusicxmlExportBreaks::MANUAL) {
-                  if (currentSystem == NewSystem && prevMeasLineBreak)
+                  if (currentSystem == NewSystem && (prevMeasLineBreak || prevMeasSectionBreak))
```

Comment:
@lvinken please review this, maybe there's a better method to export section breaks to MusicXML rather than just treat them as system breaks?

Reply:
Compiler complained about set but unsed variable. I thint the above usage is the least we can do to keep the layout, unless MusicXML does have other or additional means to denote a section break, in which case we could do better than this

Reply:
MusicXML does not support the section break concept. System break is the concept most like a section break.

Your change is correct and appreciated.

Reply:
Thanks for check and confirmation